examples/keystatic/app/[slug]/layout.tsx

```diff
@@ -0,0 +1,9 @@
+import '../styles.css'
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode
+}) {
+  return <>{children}</>
```

Comment:
You need `html` and `body` here.

Reply:
isnt that inherited from the root layout?

Reply:
Oops, I read this wrong as the root layout. Correct!